pkg/operator/ceph/object/objectstore.go

```diff
@@ -632,8 +632,8 @@ func getObjectStores(context *Context) ([]string, error) {
 	return r.Realms, nil
 }
 
-func deletePools(ctx *Context, spec cephv1.ObjectStoreSpec, lastStore bool) error {
-	if emptyPool(spec.DataPool) && emptyPool(spec.MetadataPool) {
+func DeletePools(ctx *Context, metadataPool, dataPool cephv1.PoolSpec, lastStore bool) error {
+	if emptyPool(dataPool) && emptyPool(metadataPool) {
```

Comment:
i see this just below:
```go
	if lastStore {
		pools = append(pools, rootPool)
```
we need to make sure that `rootPool=".rgw.root"` never gets deleted in a multisite setup. that's the pool that stores all of the zone/zonegroup/realm/period objects

Reply:
sure I will change this

Reply:
@cbodley : [Update] for multisite the lastStore is always set `false`, so the `.rgw.root` won't be deleted for multiistie scenario. But if only `cephobjectstores` are configured then deleting the last store ends up removing `.rgw.root` as well. Do we need to change that logic?